test/run_tests.sh

```diff
@@ -71,6 +71,7 @@ function run_all_tests {
   run_test python3 "$CDIR/test_mp_mesh_reduce.py"
   run_test python3 "$CDIR/test_xla_dist.py"
   run_test python3 "$CDIR/test_profiler.py"
+  run_test python3 "$CDIR/test_train_mp_mnist_amp.py"
```

Comment:
chat with Ailing and Daniel offline, I think we can remove this for now

Reply:
Done.